### ISSUE TITLE
Problem with [nw] command and Question event.

### ISSUE THREAD

**Issue by idontkillcoyotes** [coppolaemilio/dialogic#373]
## The problem
When using [nw] command on dialogs before a Question event with options, if the user skips the lines with the [nw] there's a chance that the options for the question get duplicated or hidden.

**Describe the bug**
As explained above, the [nw] does not work correctly if the lines are skipped, making the next events in the timeline not work as intended

**To Reproduce**
1. Create a timeline and add text event with some lines ending in [nw] at the beggining
2. Add a question, (or any other event that waits for user input)
3. Play the scene normally and skip all the lines with the [nw] command
4. See how the question options get hidden or duplicated

**System (please complete the following information):**
 - Dialogic Version 1.2
 - Godot Version: 3.2.3

## Solutions
Changing the _on_text_completed() method like this: 
```gdscript
func _on_text_completed():
    finished = true
    if current_event.has('options'):
        for o in current_event['options']:
            add_choice_button(o)
    if current_event.has('text'):
        # [p] needs more work
        #if '[p]' in current_event['text']: 
        #    yield(get_tree().create_timer(2), "timeout")
        if '[nw]' in current_event['text']:
            var current_index = dialog_index
            yield(get_tree().create_timer(2), "timeout")
            if dialog_index == current_index:
                _load_next_event()
```
Thanks to Rab_Loe#4334 on discord for this piece of code.

This change was already tested, but more testing never hurts

**Comment by Jowan-Spooner** [comment]
Thanks for writing it down, but the fix was actually already merged into 1.3 WIP along with more [nw] improvements. Thanks again for testing and finding this and sorry you had trouble during the game jam. (The game was nice anyways ;])

**Comment by coppolaemilio** [comment]
Should I close this one?
